packages/flex-plugin-scripts/src/scripts/deploy.ts

```diff
@@ -234,6 +239,7 @@ export const _doDeploy = async (nextVersion: string, options: Options): Promise<
 
   // Register service sid with Config service
   await progress('Registering plugin with Flex', async () => {
+    const configurationClient = new ConfigurationClient(credentials);
```

Comment:
Remove - keep the one from line 170. Let's initialize all clients together - cleaner to keep them together